stablehlo/reference/Interpreter.cpp

```diff
@@ -31,26 +31,24 @@ namespace stablehlo {
 
 llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
                                          ArrayRef<Tensor> args) {
-  if (func->getNumRegions() != 1) {
```

Comment:
If we're standardizing on this style guideline (which is not a bad idea), why limit ourselves to just these files?

Reply:
You're right. We should do this for all our C++ files. I've created a separate PR #1042 to address this.